### DISCUSSION TITLE
Galeria de projetos em ReactJS

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1830] [Perguntas e Respostas]
Hello Devs!!!

Estou trabalhando em meu portfolio pessoal em React e estou querendo fazer uma galeria dos meu projetos.
Encontrei um modelo de cartão para expor meus projetos, porem não tenho conhecimento de como fazer e não encontro nenhum guia.

segue o link do modelo do cartão:
https://www.4shared.com/photo/Lt5D1Kc4iq/Capturar__1_.html

Se alguem souber de algum guia de como fazer do zero eu agradeço pois só encontro já pronto e estou querendo aprender como fazer em React.

**Comment by deppbrazil** [comment]
@Al3xFlores acho que isso pode te dar uma luz,
 
* https://material-ui.com/pt/components/cards/

Coloca em expandir código em `<>` e vai ver a estrutura do componente.

**Comment by vitorfrota** [comment]
Confesso que não conheço nenhum guia que ensine isso, mas posso te ajudar 😁
